.travis.yml

```diff
@@ -14,25 +16,26 @@ services:
 
 matrix:
    include:
-        - python: 3.6
+        - python: 3.7
```

Comment:
I don't think we should remove py3.6 jobs. For starters, it's better to add new ones.